credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -70,23 +68,29 @@ def analyze(self, discovery):
 
         # Convert data format
         data = self._pre_process(raw_data)
+        # Classify as a 'Leak' if this is a private key.
+        if(self._check_private_key(data)):
```

Comment:
no () appended to the `if`. This should have been already raised by @pep8speaks, but it didn't.